tests/virt_autotest/lib/teststepapi.pm

```diff
@@ -0,0 +1,202 @@
+# SUSE's openQA tests
```

Comment:
We put this file as a basic library file, which other product can inherit it or extend functions. So that is why you may see that some function is same as in file's.